LightBlue.WebService/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Formo;
+using LightBlue.WebHost;
+using Topshelf;
+
+namespace LightBlue.WebService
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+           return (int)HostFactory.Run(x =>
+           {
+               dynamic configuration = new Configuration();
+               WebHostSettings settings = configuration.Bind<WebHostSettings>();
+
+               x.Service<WebHostService>(service =>
+               {
+                   service.ConstructUsing(s =>
+                   {
+                       var hostArgs = WebHostArgs.ParseArgs(new[]
+                       {
+                           "-a:" + settings.Assembly,
```

Comment:
I'm pretty sure it uses the -a: assembly (path to role assembly with all dlls)  and -c: configuration path (path to .cscfg) so it doesn't need to make any assumptions on other file paths.